doc/getting-started.md

```diff
@@ -654,18 +740,43 @@ Open the configuration and go to the **SCM** section. This section is available
 * Enter the SSH Git URL for the rundeck-projects repository, .
 If you use the Github repository for the rundeck-projects no further configuration is needed.
 If you use an own repository you have to enter valid authorization credentials, stored in Rundeck's key storage.
+* Branch: Choose "production" (or master for bleeding edge)
```

Comment:
I think this is not quite appropriate. My understanding is to always choose `production`. The idea of that branch is not so much stability, but rather control. The user should always decide what the content of production is, but ODS is just configured to use this branch. Even if the content of production is 100% identical to master.